apps/docs/src/components/home/Home.tsx

```diff
@@ -11,10 +11,22 @@ export const Home = () => (
     )}
   >
     {/* Background grid */}
-    <div className="absolute h-full w-full bg-[radial-gradient(var(--uikit-colors-secondary\\_80),transparent_1px)] [background-size:20px_20px] [mask-image:radial-gradient(ellipse_50%_50%_at_50%_50%,#000_70%,transparent_100%)]" />
+    <div
+      className={clsx(
+        "absolute h-full w-full",
+        "bg-[radial-gradient(var(--uikit-colors-secondary\\_80),transparent_1px)]",
+        "[background-size:20px_20px]",
+        "[mask-image:radial-gradient(ellipse_50%_50%_at_50%_50%,#000_70%,transparent_100%)]",
+      )}
```

Comment:
I'm commenting this because I see the inclusion of `clsx` as a "cope" to deal with the complexity of this class.

We could instead just use style, which is easier to understand (and don't scare away new utility class users 😅).

```tsx
className="absolute h-full w-full"
style={{
  background: `radial-gradient(${theme.colors.secondary_80},transparent 1px)`,
  backgroundSize: "20px 20px",
  maskImage: `radial-gradient(ellipse 50% 50% at 50% 50%, #000 70%, transparent 100%)`,
}}
```

Reply:
removed from the scope of this PR